.nuspec/Microsoft.Maui.Resizetizer.targets

```diff
@@ -306,9 +316,15 @@
 
         <WriteLinesToFile
             File="$(_MauiSplashInputsFile)"
-            Lines="@(MauiSplashScreen->'File=%(Identity);Link=%(Link);BaseSize=%(BaseSize);Resize=%(Resize);TintColor=%(TintColor);Color=%(Color);ForegroundScale=%(ForegroundScale)')"
+            Lines="@(_MauiSplashScreensWithHashes->'File=%(Identity);Link=%(Link);BaseSize=%(BaseSize);Resize=%(Resize);TintColor=%(TintColor);Color=%(Color);ForegroundScale=%(ForegroundScale);FileHash=%(FileHash)')"
```

Comment:
We may want to do something like this for app icons too. See #10949

Reply:
Since AppIcons changes also only seem to update on iOS only after rebooting the device, I can second this.

Reply:
Going to do that separately as this is a bit more special. Since you can select icons at runtime, you need to know the name deterministically. And the way we have it now is semi-random as the hash is based on a bunch of settings.